src/include/gurt/debug.h

```diff
@@ -1,5 +1,7 @@
 /*
  * (C) Copyright 2017-2023 Intel Corporation.
+ * (C) Copyright 2025 Google LLC
```

Comment:
why are you adding the google copyright?

Reply:
one of the commits was from me so this one makes sense